### ISSUE TITLE
Devil Revive ruined by round end

### ISSUE THREAD

**Issue by Arrfur** [BeeStation/BeeStation-Hornet#520]
Devils are supposed to be able to revive (in a similar way to lings) if the correct procedures are not taken. However, due to the fact that the devil game mode is one of the ones that instantly ends if the antagonist dies, this is impossible. Furthermore, I dont think any of the rounds should force end on the MRP server, as Sage round 7706 was ended prematurely by the devil dying on lavaland, suppose to be able to revive. The players had no input in their round coming to a jarring end.

TL;DR The devil round ending when devil dies breaks their revive mechanic and MRP rounds

**Comment by zeskorion** [comment]
Actually
I dont think devil should be on at all. It aint happend to me, but it's totally possible for your objective to just go "nah fam fuck off"

**Comment by ghost** [comment]
devils only revive if they have souls stored. If he died on lavaland with no souls thr round ends normally, not prematurely. its like a wizard roundstart killing itself

**Comment by Arrfur** [comment]
He had 4 souls stored, they dont get a chance to revive, because they have to "die" first

**Comment by Arrfur** [comment]
The devil just happened to die on lavaland, resulting in the crew having their round abrubtly (?) end

**Comment by yorii** [comment]
Killing the devils doesn't seem to end the round anymore?
